protostar-cli/src/lib.rs

```diff
@@ -80,3 +82,15 @@ pub fn get_account<'a>(
 
     Ok(account)
 }
+
+pub fn get_block_id(value: &str) -> Result<BlockId> {
+    // todo: add more block ids (hash, number)
```

Comment:
Should we create an issue for this?

Reply:
50/50

Reply:
https://github.com/software-mansion/protostar/issues/2119